symbols/overlay29.yml

```diff
@@ -8536,6 +8536,75 @@ overlay29:
         Called when the in-dungeon "others" menu is open. Does not return until the menu is closed.
         
         return: Always 0
+    - name: CheckVariousStatuses
+      address:
+        EU: 0x23016DC
+        NA: 0x2300AD0
+        JP: 0x2301D50
+      description: |-
+        Returns 0 if none of these conditions holds for the given entity: asleep, frozen, petrified, biding.
+
+        r0: Entity pointer
+        return: bool
+    - name: CheckVariousStatuses2
+      address:
+        EU: 0x2301244
+        NA: 0x2300638
+        JP: 0x23018B8
+      description: |-
+        Returns 0 if none of these conditions holds for the given entity: 
+        blinded (checked only if blind_check == 1), 
+        asleep, paused, infatuated, wrapping, wrapped, biding, petrified, or terrified.
+
+        r0: Entity pointer
+        r1: If true, return 1 if entity is blinded
+        return: bool
+    - name: IsChargingAnyTwoTurnMove
+      address:
+        EU: 0x2325084
+        NA: 0x2324478
+        JP: 0x23256F8
+      description: |-
+        Returns a boolean indicating whether or not the given entity is charging any two-turn move.
+
+        r0: Entity pointer
+        r1: Unused boolean
```

Comment:
Are you sure this is unused? Looking at the code, I see its value being checked in a conditional.

Reply:
In [Rescue Team](https://github.com/pret/pmd-red/blob/master/src/charge_move.c#L116), if this flag is true, the function will also return true if the Pokémon has the Charge status (i.e., the Electric move that boosts other Electric moves).

Reply:
This function goes through an array which contains all of the values of the bide status for which it should return true. The charge status is not listed in the array. The conditional will only be activated if iVar1 (the current index in the array) is > 99, and since the loop breaks upon reaching a 0 value in the array, it will never be activated as the first 0 occurs much sooner.

Reply:
Aha, good point. So the bool is _supposed_ to include the Charge status but it doesn't actually work as intended.

Reply:
Can we document this? Let's name the variable `charge_check_unused` or something, and then mention in the description what it's supposed to do, and how it's unused.